codeready-toolchain/toolchain-e2e
# DO NOT MERGE: testing new ci

Testing new CI:
- change to ignored files should not trigger e2e tests
- change to other files (go files) should trigger e2e tests
